src/pages/collection/[id].vue

```diff
@@ -26,5 +26,5 @@ onMounted(() => {
       Loading...
     </div>
   </WithNavbar>
-  <IconSet v-else :collection="collection" />
+  <IconSet v-else />
```

Comment:
Unrelated, I just fixed it because I noticed. This is not consumed by the component and actually renders as attribute:

![IMAGE 2022-07-26 17:21:25](https://user-images.githubusercontent.com/44045911/180972165-17caefa0-bd07-4f0b-b4e2-feae289fc616.jpg)